node/resolvers/search/modules/metadata.ts

```diff
@@ -22,10 +21,9 @@ import { formatTranslatableProp, translateManyToCurrentLanguage, Message, should
 
 type TupleString = [string, string]
 
-const isTupleMap = compose<TupleString, string, boolean>(
-  equals('c'),
-  prop('1')
-)
+const categoryKeys = ['c', 'category', 'category-1', 'category-2', 'category-3']
+
+const isTupleMap = (t: TupleString) => t[1] ? categoryKeys.includes(t[1]) : false
```

Comment:
```suggestion
const isTupleMap = (t: TupleString) => categoryKeys.includes(t[1])
```
In case t[1] is undefined, the output is`false`